deeprankcore/utils/graph.py

```diff
@@ -99,6 +99,9 @@ def __init__(self, id_: str):
         # targets are optional and may be set later
         self.targets = {}
 
+        # the center only needs to be set when this graph should be mapped to a grid.
+        self.center = np.array((0.0, 0.0, 0.0))
+
```

Comment:
Then why does the Graph class need a center attribute?

Reply:
That's right! They will be overwritten once the center is known. The problem is that this center position can only be determined from the atoms in the structure. The method `write_as_grid_to_hdf5` only takes a graph as input, not a structure. So I chose to store the center in the graph, just like the node positions.

I agree that this is not a clean solution. Should we redesign the class-diagram?